nextstrain/cli/command/run.py

```diff
@@ -276,11 +300,15 @@ def run(opts):
 
         # Workdir will be the analysis volume (/nextstrain/build in a
         # containerized runtime), so explicitly point to the Snakefile.
-        "--snakefile=%s/Snakefile" % (
-            docker.mount_point(workflow_volume)
-                if opts.__runner__ in {docker, singularity, aws_batch} else
-            workflow_volume.src.resolve(strict = True)),
+        "--snakefile=%s" % (resolved_snakefile),
+
+        *(["--configfile=%s" % (resolved_configfile)]
+            if resolved_configfile else []),
 
+        # Ensure the overlay config in the user's analysis directory
+        # overrides any default config file provided above.
+        *(["--configfile=%s" % (resolved_overlay)]
+            if resolved_overlay else []),
```

Comment:
I'm confused... the way things are worded in the `nextstrain run` [changelog entry](https://github.com/nextstrain/cli/blob/f04761d3b2a89620ccd3c00fc44591762ab742a5/CHANGES.md#L143-L144) makes it sound like `<analysis dir>/config.yaml` is auto-applied, however I don't see any indication of that in the code. It seems like this change is finally making that happen?

Reply:
Yeah, it wasn't built into the `nextstrain run` command. The command relied on the underlying Snakemake workflow to use the config file in the analysis directory via the `configfile:` statement (e.g. measles' [ingest Snakefile](https://github.com/nextstrain/measles/blob/02ee7f1e9b99d0dbe9f1da63f06a57149b94c201/ingest/Snakefile#L12-L14)). That no longer worked here since `--configfile` overrides the `configfile:` in the Snakefile, so I added this change (see detailed [discussion above](https://github.com/nextstrain/cli/pull/462#discussion_r2226909561)).

Reply:
Gotcha, it makes sense now with the measles example. Thanks!